drivers/SmartThings/jbl/src/jbl/device_manager.lua

```diff
@@ -0,0 +1,144 @@
+local log = require "log"
+local json = require "st.json"
+local fields = require "fields"
+
+local capabilities = require "st.capabilities"
+
+local device_manager = {}
+device_manager.__index = device_manager
+
+local function is_new_audioTrackData(device, audioTrackData)
+    local latestTrackData = device:get_latest_state("main", capabilities.audioTrackData.ID, capabilities.audioTrackData.audioTrackData.NAME, "")
+
+    if not latestTrackData or latestTrackData == "" then
+        return true
+    end
+
+    if (audioTrackData.title == latestTrackData.title)
+        and (audioTrackData.artist == latestTrackData.artist)
+        and (audioTrackData.album == latestTrackData.album)
+        and (audioTrackData.albumArtUrl == latestTrackData.albumArtUrl)
+        and (audioTrackData.mediaSource == latestTrackData.mediaSource) then
+        return false
+    end
+
+    return true
+end
+
+local jbl_playback_state_to_smartthings_playback_status_table = {
+    paused = "paused",
+    playing = "playing",
+    stopped = "stopped",
+}
```

Comment:
This doesn't appear to be mapping between anything?

Reply:
Looking further on, it seems like a "set" table here would be enough.

```suggestion
local valid_playback_status = {
    paused = true,
    playing = true,
    stopped = true
}
```

```lua
if not valid_playback_status[status.playback] then
  return
end
```